File: desktop/src/be/ucll/dirkfalls/desktop/DesktopLauncher.java
package be.ucll.dirkfalls.desktop;

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;

import be.ucll.dirkfalls.GameConfig;
import be.ucll.dirkfalls.TestGame;

public class DesktopLauncher {
    public static void main(String[] arg) {
        LwjglApplicationConfiguration config = new LwjglApplicationConfiguration();
        config.width = GameConfig.WIDTH;
        config.height = GameConfig.HEIGHT;

        //new LwjglApplication(new DirkFallsGame(), config);
        //new LwjglApplication(new ShapeRenderSample(), config);
        new LwjglApplication(new TestGame(), config);
    }
}

File: desktop/src/be/ucll/dirkfalls/desktop/DesktopLauncher.kt
package be.ucll.dirkfalls.desktop

import be.ucll.dirkfalls.DirkFallsGame
import com.badlogic.gdx.backends.lwjgl.LwjglApplication
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration

fun main(arg: Array<String>) {
    val config = LwjglApplicationConfiguration().apply {
        width = 480
        height = 800
    }
    val fb = FacebookDesktop()
    LwjglApplication(DirkFallsGame(fb), config)
}
